arch/arm/boot/dts/overlays/README

```diff
@@ -4074,7 +4074,8 @@ Params: speed                   SPI bus speed (default 10000000)
         dc_pin                  GPIO pin for D/C (default 24)
         reset_pin               GPIO pin for RESET (default 25)
         height                  Display height (32 or 64; default 64)
-
+        inverted                Set this if display is inverted and mirrored.
+                                (default=not set)
 
```

Comment:
Unfortunately you've deleted one of the two required blank lines between entries, as the error log says:
```
* Missing blank lines after overlay? (4080)
```

Reply:
Ups, sorry about that. It should be fixed now.